ingestion_server/ingestion_server/cleanup.py

```diff
@@ -300,6 +308,10 @@ def clean_image_data(table):
     :return: None
     """
 
+    # Recreate directory where cleaned data is stored
+    shutil.rmtree(TMP_DIR)
```

Comment:
TIL the difference between `Path::rmdir` and `shutil.rmtree`. The former can only remove empty directories (the bit I didn't know!).

Reply:
It's a striking difference but I guess internally it should make sense to keep functions with unique responsibilities.

Reply:
For sure!